_src/app/App.js

```diff
@@ -82,20 +74,30 @@ define([
         // mapView: map-tools/MapView
         mapView: null,
 
-        constructor: function () {
+        //      used to preserve the current extent between sessions
+        localStorageKeys: {
+            zoom: 'agrc-atlas-current-zoom',
+            center: 'agrc-atlas-current-center'
+        },
+
+        constructor() {
             // summary:
             //      first function to fire after page loads
-            console.info('app.App::constructor', arguments);
+            console.log('app/App:constructor', arguments);
 
             config.app = this;
             this.childWidgets = [];
 
             this.inherited(arguments);
         },
-        postCreate: function () {
+        postCreate() {
             // summary:
             //      Fires when
-            console.log('app.App::postCreate', arguments);
+            console.log('app/App:postCreate', arguments);
+
+            if (window.innerWidth < config.minWidthToShowSidebarOnLoad) {
```

Comment:
this isn't really a good responsive way to do this is it?

Reply:
This is only to affect on how the app loads. I still want to be able to open the sidebar on smaller screens. How would we do that via media queries? Some sort of override class?